torch/nn/quantized/modules/conv.py

```diff
@@ -204,7 +204,9 @@ def get_qconv(cls, mod, activation_post_process, weight_post_process=None):
         qconv = cls(mod.in_channels, mod.out_channels, mod.kernel_size,  # type: ignore[call-arg]
                     mod.stride, mod.padding, mod.dilation, mod.groups,
                     mod.bias is not None, mod.padding_mode)
-        qconv.set_weight_bias(qweight, mod.bias)
+        # TODO(before land): also set this for all other conv modules and functionals
+        activation_safe_on_fbgemm = activation_post_process.reduce_range
```

Comment:
We have been deprecating the use of reduce_range attribute in observers, so we need to infer this based on quant_min and quant_max instead.

Reply:
This might have to be in a future PR.  In the current code `self.reduce_range` is the only persistent state available in observers for default settings, and `self.quant_min` and `self.quant_max` are only persisted if the user is using customized ranges.